source/__tests__/integration/home-page.test.js

```diff
@@ -20,11 +20,12 @@ describe('Sinopia Profile Editor Homepage', () => {
   })
 
   describe('header', () => {
-    it('image', async() => {
-      await expect_sel_to_exist('div.sinopia-headertext > img[src="assets/images/sinopia_profile_headertext.png"]')
```

Comment:
this image went away?  for consistency with links?

Reply:
The design changed in #146 (header spans two lines instead of one)